concepts/vector.md

```diff
@@ -1,7 +1,33 @@
 ---
 grand_parent: Resources
 parent: Concepts
-layout: coming_soon
 title: Vector
 description:
 ---
+
+A **vector** is a unique fixed-sized list of numbers.
+Each vector number is called coordinate.
+Coordinates define the vector's position in a particular space.
+Vectors represent data in a way that a machine learning model can understand and use.
+In machine translation, vectors represent the words and phrases in a text.
+
+Example of a vector:
+
+`cat` = [4564]
```

Comment:
I think our example should be at least length 3, or even include ellipses.

Reply:
Trust